blint/analysis.py

```diff
@@ -545,7 +545,7 @@ def _review_symbols_exe(self, metadata):
         symbols_list = [f.get("name", "") for f in
                         metadata.get("dynamic_symbols", [])]
         symbols_list += [f.get("name", "") for f in
-                         metadata.get("static_symbols", [])]
+                         metadata.get("symtab_symbols", [])]
```

Comment:
@prabhu I thought this would not change until lief 0.15.0, which has not been released yet..?

Reply:
I think this is fine. Static symbols is incorrect so we can use symtab symbols or just symbols

Reply:
@prabhu But we are trying to get this from lief generated data, no? I understand static symbols is incorrect in terms of the accuracy of lief naming and that's why it will change in 0.15.0 - but is it not still static symbols in the 0.14.1 API?

The [commit](https://github.com/lief-project/LIEF/commit/cddd8dd0768334f341242c8f61e0819e8fc4f885) which changed the naming is listed among the [commits made to master](https://github.com/lief-project/LIEF/compare/0.14.1...master) since 0.14.1. Therefore until we can bump to 0.15.0, we will not get any data using symtab_symbols.

Reply:
@cerrussell, since we are surfacing this information via SBOM, it is best to use symtab_symbols as the property name to avoid a breaking change. We can change this [line](https://github.com/owasp-dep-scan/blint/blob/feature/generic-binary-sbom/blint/binary.py#L721) once 0.15.0 gets released.